src/components/TemplateContent.vue

```diff
@@ -0,0 +1,221 @@
+<template>
+  <div :class="'blip-container template-content ' + position">
+    <div :class="'bubble ' + position">
+      <div class="template-content-header">
+        <bds-icon v-if="position === 'right'" size="small" color="white" alt="paperplane" name="paperplane"></bds-icon>
+        <bds-icon v-else size="small" alt="paperplane" name="paperplane"></bds-icon>
+        <bds-typo :class="'typo template-content-text-body ' + position" variant="fs-16" bold="semi-bold">{{ showTemplateContentTitle }}</bds-typo>
+      </div>
+      <div class="template-content-text flex flex-column">
+        <bds-typo class="span" variant="fs-16" bold="semi-bold"  v-for="(link, index) in showTemplateContentLinks()" :key="index">
+          <a :href="link" target="_blank">{{ link }}</a>
+        </bds-typo>
+        <bds-typo :class="'span template-content-text-body ' + position" variant="fs-16" bold="regular" v-html="formatText(showTemplateContentBody(), 'template-content-text-body ' + position)"/>
+      </div>
+    </div>
+    <bds-icon v-if="this.position === 'right' && this.status === 'failed' && this.onFailedClickIcon"
+        name="info" theme="solid"
+        aria-label="Active message failed reason"
+        class="icon-active-message-failed"
+        @click="onFailedClickIcon(fullDocument)"></bds-icon>
+    <div class="flex" :class="'notification ' + position" v-if="date">
+      <span v-if="this.position === 'right'">
+        <img v-if="this.status === 'waiting'" :src="clockSvg">
+        <img v-else-if="this.status === 'accepted'" :src="checkSentSvg">
+        <img v-else-if="this.status === 'received'" :src="doubleCheckReceivedSvg">
+        <img v-else-if="this.status === 'consumed'" :src="doubleCheckReadSvg">
+        <div v-else-if="this.status === 'failed'" class="failure">
+          {{ failedToSendMsg }}
+        </div>
+      </span>
+      <div>{{ date }}</div>
+    </div>
```

Comment:
I think you can replace this with the `BlipCardDate` component.

Reply:
Thanks for the suggestion!